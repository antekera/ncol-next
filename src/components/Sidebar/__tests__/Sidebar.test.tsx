import React from 'react'

import { render } from '@testing-library/react'

import { Sidebar } from '..'

describe('Sidebar', () => {
  test('should match snapshots', () => {
    const { container } = render(<Sidebar />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
