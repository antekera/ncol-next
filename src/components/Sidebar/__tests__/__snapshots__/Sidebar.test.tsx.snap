// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should match snapshots 1`] = `
<aside
  class="w-full px-2 md:w-1/3 lg:w-1/4"
>
  <div
    class="mb-8 rounded-lg bg-slate-100 p-4 font-sans md:mb-4 hidden md:block"
  >
    <form
      action="javascript:throw new Error('A React form was unexpectedly submitted. If you called form.submit() manually, consider using form.requestSubmit() instead. If you\\'re trying to use event.stopPropagation() in a submit event handler, consider also calling event.preventDefault().')"
    >
      <label
        class="mb-0 pt-1 font-sans_bold text-lg"
        for="email-input"
      >
        Únete a nuestro boletín
      </label>
      <p
        class="text-sm leading-snug"
      >
        Recibe grátis las noticias más destacadas en tu correo.
      </p>
      <div
        class="mt-2 flex flex-col"
      >
        <div
          class="w-full"
        >
          <input
            autocapitalize="off"
            autocorrect="off"
            class="shadow-sm focus:darkBlue mb-2 block w-full rounded-md border border-darkBlue/20 px-3 py-1 text-sm focus:ring-opacity-50 md:w-11/12"
            id="email-input"
            name="email"
            placeholder="tu.correo@mail.com"
            required=""
            type="email"
            value=""
          />
        </div>
        <div
          class="w-1/5 md:w-full"
        >
          <button
            class="focus:shadow-outline flex gap-2 rounded-lg border bg-primary px-2 py-1 text-sm text-white transition-colors duration-150 hover:bg-darkBlue/80 md:px-3"
          >
            <svg
              class="lucide lucide-mail"
              fill="none"
              height="20"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                height="16"
                rx="2"
                width="20"
                x="2"
                y="4"
              />
              <path
                d="m22 7-8.97 5.7a1.94 1.94 0 0 1-2.06 0L2 7"
              />
            </svg>
            Suscribirme
          </button>
        </div>
      </div>
    </form>
  </div>
  <div
    class="show-desktop"
  />
</aside>
`;
